app/Helpers/Slack.php

```diff
@@ -8,18 +8,14 @@
 
 namespace App\Helpers;
 
+use Illuminate\Support\Facades\Log;
+
 class Slack
 {
 
     public static function sendInvitation($email) {
-        //return \Unirest\Request::post(config('slack.invitation_url'), [], ['token' => config('slack.legacy_token'), 'email' => $email]);
-        $client = new \GuzzleHttp\Client();
-        $response =  $client->request('POST', config('slack.invitation_url'), ['query' => [
-            'token' => config('slack.legacy_token'),
-            'email' => $email
-        ]]);
-        $response = json_decode($response->getBody());
-        return $response;
+        Log::info($email . ' attempted to obtain an invitation to Slack');
```

Comment:
IP address as well?

Reply:
I think for now, Logging the email is sufficient as sometimes IP addresses might be VPN based.

Reply:
While there's nothing to stop us storing IP addresses, some do think it counts as PII, as discussed in Slack, but only when stored against another piece of PII.

Emails are more valuable to us than IP's at this point, so we just store those for now in the log. :)